docs/src/main/sphinx/release/release-403.md

```diff
@@ -0,0 +1,38 @@
+# Release 403 (9 Nov 2022)
+
+## General
+
+* Round `TimeWithTimeZone` values when `PARAMETRIC_DATETIME` is not set. ({issue}`14950`)
+* Fix query deadlock in multi-join queries where broadcast join size is
+  underestimated. ({issue}`14948`)
+* Fix incorrect results when `min(x, n)` or `max(x, n)` is used as a window
```

Comment:
do we need to detail what was wrong before?

Reply:
I don't think we need to spell it out here. If readers want more details, they can navigate to the issue.